bentoml/_internal/service/service.py

```diff
@@ -37,7 +37,6 @@ class Service:
     """
 
     _apis: t.Dict[str, InferenceAPI] = {}
```

Comment:
Should we just make `_apis` also public, same as runners?

Also setting the default to `{}` was a huge mistake by me 😂 we should change that in a separate PR